releases/nightly-build/compose-files/README.md

```diff
@@ -0,0 +1,44 @@
+## Edgex Docker Compose for `master` builds
+
+This folder contain the docker compose files that pull and run the EdgeX images from the Nexus3 docker registry that are tagged `master`. These images are build from the Edgex CI Pipeline when PRs are merged into the `master` branch.
+
+> *Note: Docker does not re-pull newer instances of these images. You must explicitly remove old instances so the new instances are pulled. See the `make remove` command below that will do this for you.*
```

Comment:
Why not just have a rule that does the docker-compose pull?

Reply:
I considered that, but `pull` which config? Pull for all configs, isn't efficient? Didn't want a buch of `pull` variations. This makes sure when you run your config the images it uses are freshly pulled.

Reply:
Ah, I see.  Perhaps a reasonable middle ground would be to pull all containers per arch, whether or not they are actually run.  Or maybe a makefile isn't the right option here.  Maybe it should just be a shell script that simply computes the .yml files needed based on command line options, computes the architecture by scrutinizing the output of uname -a, and then passes additional flags such as up, down, pull, etc to docker-compose?